src/main/java/uk/co/ramp/io/ContactReader.java

```diff
@@ -0,0 +1,41 @@
+package uk.co.ramp.io;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Service;
+import uk.co.ramp.event.types.ContactEvent;
+import uk.co.ramp.event.types.ImmutableContactEvent;
+import uk.co.ramp.io.csv.CsvReader;
+import uk.co.ramp.io.types.StandardProperties;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class ContactReader {
+    private static final Logger LOGGER = LogManager.getLogger(ContactReader.class);
+
+    public Map<Integer, List<ContactEvent>> readEvents(Reader reader, StandardProperties properties) throws IOException {
```

Comment:
```suggestion
    private final StandardProperties properties;
    @Autowired
    public ContactReader(StandardProperties standardProperties) {
        this.properties = standardProperties;
    }

    public Map<Integer, List<ContactEvent>> readEvents(Reader reader) throws IOException {
```
We could avoid passing in standardproperties (which is pretty much static, does not change per method call) by passing it through constructor and injecting it via Spring